examples/trials/network-morphism/config.yml

```diff
@@ -0,0 +1,25 @@
+authorName: default
+experimentName: example_cifar10-network-morphism
+trialConcurrency: 4
+maxExecDuration: 48h
+maxTrialNum: 200
+#choice: local, remote, pai
+trainingServicePlatform: local
+#searchSpacePath: search_space.json
+#choice: true, false
+useAnnotation: false
+tuner:
+  #choice: TPE, Random, Anneal, Evolution, BatchTuner, Medianstop, NetworkMorphism
+  #SMAC (SMAC should be installed through nnictl) 
+  builtinTunerName: NetworkMorphism
+  classArgs:
+    #choice: maximize, minimize
+    optimize_mode: maximize
+    task: cv
```

Comment:
I think we could add "# for now, we only support cv domain"

Reply:
ok